backend/start_app.sh

```
@@ -0,0 +1,15 @@
+
+#!/bin/sh
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)